app/furniture/marketplace/carts/_cart.html.erb

```diff
@@ -16,7 +16,7 @@
         </tr>
       </thead>
       <tbody class="divide-y divide-gray-200 bg-white">
-        <%- cart.marketplace.products.each do |product| %>
+        <%- cart.marketplace.products.kept.each do |product| %>
```

Comment:
Is `cart.marketplace.products` different from `cart.marketplace_products`? Is there a reason to use one or the other?

Reply:
There is no `cart.marketplace_products`. There is `cart.products`, which is the products associated to that particular `cart` and there is `cart.marketplace.products` which is the `products` that belong the the `marketplace` that the `cart` is part of.

Reply:
Totally subjective commentary: for some reason `kept` feels like a weird word to me to see in code, idk why.

Reply:
> There is no `cart.marketplace_products`. There is `cart.products`, which is the products associated to that particular `cart` and there is `cart.marketplace.products` which is the `products` that belong the the `marketplace` that the `cart` is part of.

Oh I saw `cart.cart_products` in the code elsewhere and misread it.

Reply:
Agreed that `kept` is somehow... funny? I believe the discard gem also offers `undiscarded` as an alias to the `kept` predicates and scopes, so maybe we should use that instead, since it is anyway clearer.

FWIW, on a client project where we use this gem we alias the `discard`/`undiscard` methods and scopes to a name that matches semantically whatever we are doing on that particular model. So e.g. on an accounting model that represents a charge to a client, I aliased `discard` to be `waive`. We could do the same here and make scopes `archived`, `unarchived`, etc.

Reply:
I like that, I kinda maybe went overboard by renaming everything.

Reply:
Aliasing sounds like a good idea since it's more domain-driven.